Sources/TuistKit/Services/ShareService.swift

```diff
@@ -378,5 +390,15 @@ struct ShareService {
                 "preview_id": "\(preview.id)",
             ]
         )
+
+        if json {
```

Comment:
At some point when we dependency-inject the logger, we should inject a logger that prevents any other logs from happening causing the integration with this to break.

Reply:
> At some point when we dependency-inject the logger, we should inject a logger that prevents any other logs from happening causing the integration with this to break.

That's already the case. Only logs with `metadata: .json` are actually posted when the `--json` flag is present.